OptimoveCore/Tests/Sources/Storage/OptimoveStorageFacadeTests.swift

```diff
@@ -37,4 +38,25 @@ class OptimoveStorageFacadeTests: XCTestCase {
         let value: String? = storage[key]
         XCTAssert(value == stub_string)
     }
+
+    func test_try_get_subscript() throws {
```

Comment:
Are these tests to cover that we use the alternate storage for certain keys?

Reply:
No, this test checking a new swift subscript method which can throws an error. It was not possible a few years ago with previous versions of swift lang